crates/matrix-sdk/src/room/mod.rs

```diff
@@ -2976,6 +2979,60 @@ impl Room {
             .await?;
         Ok(())
     }
+
+    /// Subscribe to live location sharing events for this room.
+    ///
+    /// The returned receiver will receive a new event for each sync response
+    /// that contains a 'm.beacon' event.
+    pub fn subscribe_to_live_location_shares(
+        &self,
+    ) -> (JoinHandle<()>, broadcast::Receiver<LiveLocationShare>) {
+        let (sender, receiver) = broadcast::channel(16);
+
+        let client = self.client.clone();
+        let room_id = self.room_id().to_owned();
+
+        let handle: JoinHandle<()> = tokio::spawn(async move {
```

Comment:
Can you use `matrix_sdk_common::executor::spawn` instead? So that we have WebAssembly support.